_2020/version-control.md

```diff
@@ -465,7 +465,13 @@ command is used for merging.
 - `git pull`: same as `git fetch; git merge`
 - `git clone`: download repository from remote
 
-# Advanced Git
+## Undo Git
```

Comment:
Could we change this to just "Undo"?

And `git commit --amend` already appears above. If we have an undo section, this seems like a better place for it. Could you remove the duplicate in the Basics section?